README.md

```diff
@@ -70,53 +304,184 @@ Make sure you have Postgres, Redis and Vault properly installed & configured. Do
 
 > If you want to run the UI app in development mode, i.e. with HMR enabled, please follow the steps in the [Development (UI)](#development-ui) section.
 
-## Configuration
+---
 
-For a full user guide, please refer to the [getting started docs](https://0xpolygonid.github.io/tutorials/issuer-node/getting-started-flow).
+## Issuing Credentials/Claims
+
+ℹ️ **NOTE:** A lot of these steps can be done through the API UI at [http://localhost:3001](http://localhost:3001).
+
+Once you've completed the [Installation](#installation) section, this will walk you through issuing credentials/claims.
+
+### (Optional/Skip) Create Identity
+
+This is to demonstrate how to createa an identity but is not needed as the identity is created in [Create Issuer DID](#create-issuer-did) step.
+
+```bash
+# NOTE: dXNlcjpwYXNzd29yZA== is a Basic HTTP Authorizatio as base64(user-issuer:password-issuer) from our config.toml file
```

Comment:
we are no longer using the `config.toml` file

Reply:
Nice catch. I'll remove it.